api/app/signals/settings/base.py

```diff
@@ -17,15 +17,7 @@
 # Django settings
 SECRET_KEY = os.getenv('SECRET_KEY')
 DEBUG = False
-ALLOWED_HOSTS = [
-    'api.data.amsterdam.nl',
-    'acc.api.data.amsterdam.nl',
-
-    # Currently this is needed because the deployment process checks the health endpoint with a
-    # request to localhost:port.
-    'localhost',
-    '127.0.0.1',
-]
+ALLOWED_HOSTS = os.getenv('ALLOWED_HOSTS', 'api.data.amsterdam.nl,acc.api.data.amsterdam.nl,localhost,127.0.0.1').split(',')
```

Comment:
Can you leave the comments about the deployment process and why localhost is needed for Amsterdam.

Reply:
Sure, added the comment back.

Now `ALLOWED_HOSTS` is configurable it maybe makes more sense to not whitelist `localhost` and `127.0.0.1` by default.

Reply:
True but this is needed for now because of the deployment setup at datapunt

Reply:
Sure, will resolve this then for now :)